src/cloud/docker/docker-extend.md

```diff
@@ -75,6 +75,60 @@ To add a new service version using a `Dockerfile`:
 
 1. Once the build succeeds, test the changes by specifying the [Docker build sources].
 
+## Add a new PHP extension
+
+In addition to built in extensions, you can add PHP extensions to the PHP container .
+
+{:.procedure}
+To add a new PHP extension:
+
+1. Clone the `{{site.data.var.mcd}}` project to your local environment if necessary.
+
+1. On the command line, navigate to the following directory
+   ```bash
+      cd magento-cloud-docker/src/Compose/Php
+   ```
+
+1.  Open `ExtensionResolver.php` file, define the required extension inside the `getConfig` method by specifying the extension type and dependency.
+
+    For instance the following block adds the GNUPG extension:
+    ```PHP
+    'gnupg' => [
+      '>=7.0' => [
+      self::EXTENSION_TYPE => self::EXTENSION_TYPE_PECL,
+      self::EXTENSION_OS_DEPENDENCIES => ['libgpgme11-dev'],
+    ],],
+    ```
+1. The extension type can be either CORE or PECL which are defined by `EXTENSION_TYPE_PECL` or `EXTENSION_TYPE_CORE` respectively.
+
+1. If `.ini` files are to be added, do the following:
+
+   * Navigate to the following directory
+
+        ```bash
+       cd magento-cloud-docker/images/php/fpm
+        ```
+   * Add the .ini files inside `etc` directory
+   * Open `Dockerfile` and add the following line to it
+
+        ```COPY etc/<filename>.ini /usr/local/etc/php/conf.d/<filename>.ini```
+   * For CLI container, Navigate to the following directory
+
+        ```bash
+        cd magento-cloud-docker/images/php/cli
+        ```
+   * Add the .ini files inside `etc` directory
+   * Open `Dockerfile` and add the following line to it
+
+      ```ADD etc/<file-name>.ini /usr/local/etc/php/conf.d/<filename>.ini```
+
```

Comment:
```suggestion
   -  Open `Dockerfile` and add the following line to it

      ```conf
      ADD etc/<file-name>.ini /usr/local/etc/php/conf.d/<filename>.ini
      ```

```

Reply:
Hi @meker12 , thanks a lot for the suggestions, I have made all the changes and fixed the spacing issues.
Thanks.